core/services/assets/BlockAssetManager.php

```diff
@@ -155,13 +155,19 @@ public function addEditorScript($handle, array $dependencies = array())
         if($this->assets->hasJavascriptAsset($handle)){
             return $this->assets->getJavascriptAsset($handle);
         }
-        return parent::addJavascript(
+        return $this->addJavascript(
```

Comment:
these look like they could use the new helpers you added as well

Reply:
Yup, see 3cc025c